static/panes/tool.js

```diff
@@ -511,6 +515,31 @@ Tool.prototype.onCompileResult = function (id, compiler, result) {
             if (toolResult.sourcechanged && this.editorId) {
                 this.eventHub.emit('newSource', this.editorId, toolResult.newsource);
             }
+            this.artifactBtn.off('click');
+            if (toolResult.artifactGenerated) {
+                this.artifactBtn.removeClass('d-none');
+                this.artifactText.text(`Download ${toolResult.artifactGenerated.title}`);
```

Comment:
Sorry for so much ES5 releated changes, hopefully we can port this to TS soon. Until then:
```suggestion
                this.artifactText.text('Download ' + toolResult.artifactGenerated.title);
```

Reply:
Sorry, I'm so fancy with all my modern JS!